packages/ui-extensions-react/src/surfaces/customer-account/components/shared-checkout-components.ts

```diff
@@ -1,109 +1,145 @@
 export {
   Banner,
+  type BannerProps,
   BlockLayout,
+  type BlockLayoutProps,
   BlockSpacer,
+  type BlockSpacerProps,
   BlockStack,
+  type BlockStackProps,
   Button,
+  type ButtonProps,
   Checkbox,
+  type CheckboxProps,
   Choice,
+  type ChoiceProps,
   ChoiceList,
-  DateField,
+  type ChoiceListProps,
   DatePicker,
+  type DatePickerProps,
+  type DayString,
+  type DateString,
+  type DisabledDate,
+  type DateRange,
+  type SelectedDate,
+  type YearMonth,
+  DateField,
+  type DateFieldProps,
   Disclosure,
+  type DisclosureProps,
   Divider,
+  type DividerProps,
   Form,
+  type FormProps,
   Grid,
+  type GridProps,
   GridItem,
+  type GridItemProps,
   Heading,
+  type HeadingProps,
   HeadingGroup,
+  type HeadingGroupProps,
   Icon,
+  type IconProps,
+  // type IconSource,
```

Comment:
Will you uncomment these after https://github.com/Shopify/ui-extensions/pull/1420 is shipped?